src/routes/transactions/entities/__tests__/human-description.builder.ts

```diff
@@ -0,0 +1,42 @@
+import { Builder, IBuilder } from '@/__tests__/builder';
+import {
+  RichAddressFragment,
+  RichTokenValueFragment,
+  RichTextFragment,
+  RichDecodedInfoFragment,
+  RichFragmentType,
+} from '@/routes/transactions/entities/human-description.entity';
+import { faker } from '@faker-js/faker';
+
+function richTokenValueFragmentBuilder(): IBuilder<RichTokenValueFragment> {
+  return Builder.new<RichTokenValueFragment>()
+    .with('type', RichFragmentType.TokenValue)
+    .with('value', faker.number.int().toString())
+    .with('symbol', faker.finance.currencySymbol())
+    .with('logoUri', faker.internet.avatar());
+}
+
+function richWordFragmentBuilder(): IBuilder<RichTextFragment> {
+  return Builder.new<RichTextFragment>()
+    .with('type', RichFragmentType.Text)
+    .with('value', faker.word.adverb());
```

Comment:
nitpick: semantics only (no need to change) but you also have the following functions:

`faker.lorem.word()` – https://fakerjs.dev/api/lorem.html#word
`faker.word.sample()` – https://fakerjs.dev/api/word.html#sample